packages/knip/src/plugins/vitest/index.ts

```diff
@@ -45,7 +45,19 @@ const findConfigDependencies = (localConfig: ViteConfig, options: PluginOptions)
   const setupFiles = [testConfig.setupFiles ?? []].flat().map(specifier => ({ ...toDeferResolve(specifier), dir }));
   const globalSetup = [testConfig.globalSetup ?? []].flat().map(specifier => ({ ...toDeferResolve(specifier), dir }));
 
-  return [...[...environments, ...reporters, ...coverage].map(id => toDependency(id)), ...setupFiles, ...globalSetup];
+  const workspaceDependencies: Input[] = [];
+  if (testConfig.workspace !== undefined) {
```

Comment:
Please update the types for this plugin (https://github.com/webpro-nl/knip/pull/1003/checks#step:6:18)

Reply:
https://github.com/webpro-nl/knip/pull/1003/commits/27564a4a14a97a7103d604ff21e36fa9a8d1324c